ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.14.yaml

```diff
@@ -14,18 +14,23 @@ images:
   to: oadp-operator
 - dockerfile_path: build/ci-Dockerfile
   from: rhel-8-golang-1.20
-  to: test-oadp-operator
+  to: test-oadp-operator-master
 operator:
   bundles:
   - dockerfile_path: build/Dockerfile.bundle
   substitutions:
   - pullspec: quay.io/konveyor/oadp-operator:latest
     with: oadp-operator
+promotion:
+  excluded_images:
+  - oadp-operator
+  name: test-oadp-operator-master
+  namespace: konveyor
```

Comment:
although I don't know why we were excluding images here in 4.12.
Perhaps it would make sense to not exclude it.

The goal of promotion is to allow for mirroring so you could add a line for oadp-operator image to
https://github.com/kaovilai/release/blob/master/core-services/image-mirroring/konveyor/mapping_konveyor_latest

Reply:
going to remove excluded then in following commit

Reply:
then also add mirroring please. We disabled travis awhile back which stopped producing quay images iiuc.
Last push here was 7 months ago.

https://github.com/openshift/oadp-operator/issues/1228

Reply:
how I add mirroring?

Reply:
add a line here https://github.com/kaovilai/release/blob/master/core-services/image-mirroring/konveyor/mapping_konveyor_latest

Reply:
```
registry.ci.openshift.org/konveyor/oadp-operator:latest konveyor/oadp-operator:latest
registry.ci.openshift.org/konveyor/oadp-operator:oadp-1.3 konveyor/oadp-operator:oadp-1.3
registry.ci.openshift.org/konveyor/oadp-operator:oadp-1.2 konveyor/oadp-operator:oadp-1.2
registry.ci.openshift.org/konveyor/oadp-operator:oadp-1.1 konveyor/oadp-operator:oadp-1.1
registry.ci.openshift.org/konveyor/oadp-operator:oadp-1.0 konveyor/oadp-operator:oadp-1.0
```

Reply:
still not clear for me

for example: this image was build 10 minutes ago (a lot of tags, but lets focus in `latest`) https://quay.io/repository/konveyor/velero?tab=tags

This should be triggered by https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/branch-ci-openshift-velero-konveyor-dev-images/1719380192402608128 (which was triggered by last commit in default branch of https://github.com/openshift/velero, 2 weeks ago)

Reply:
Work for mirroring images to quay will be follow up

> this image was build 10 minutes ago

The mirror job in openshift-ci runs hourly, it does not line up with when commits are pushed.
When commits are pushed to branch, post submit image job promotes aka push to registry.ci.openshift
then hourly.. regardless of what's going on, the mirror job defined, will copy from registry.ci.openshift to quay.io

Reply:
> Periodically, oc image mirror is used to push a configured set of images to Quay repositories. A number of Quay repositories already have mirroring pipelines configured; each directory [here](https://github.com/openshift/release/tree/master/core-services/image-mirroring) corresponds to a repository. These directories contain mapping files that define tags on images in the target repository. New images may be submitted to mirror to existing organizations, or new ones. When naming your new image, please follow the [naming guidelines](https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md).

https://docs.ci.openshift.org/docs/how-tos/mirroring-to-quay/#mirroring-images

It's a dumb mirroring system.

Reply:
images on quay built with this system (the alternative is travis/circle-ci/github-actions, which we are not using) will always show `last modified` up to "an hour ago" due to the houly mirror job.

Reply:
ow understood, it is not a problem in our side
ok, I will add lines to mirroring. 2 questions: 
- in this PR?
- in release repo, right? or needs to be in your fork?

Reply:
For mirroring it will be in release repo

Reply:
resolving
we discussed and we will remove excluded in this PR and will open follow up PR for updating mirroring file